V9_MIGRATION.md

```diff
@@ -66,3 +66,5 @@ dropInRequest.uiCustomization = uiCustomization
 ## Drop-in Result
 
 The `isCancelled` property on `BTDropInResult` has been changed to `isCanceled`.
+
+The `paymentOptionType` property on `BTDropInResult` has been changed to `paymentMethodType`.
```

Comment:
Should we call out that the enum type name also changed?

Reply:
```suggestion
The `paymentOptionType` property on `BTDropInResult` has been changed to `paymentMethodType`. The type of this enum has changed from `BTUIKPaymentOptionType` to `BTDropInPaymentMethodType`.
```